modules/tools/lab-partner/main.tf

```diff
@@ -0,0 +1,33 @@
+data "helm_repository" "liatrio" {
+  name = "liatrio"
+  url  = "https://liatrio-helm.s3.us-east-1.amazonaws.com/charts"
+}
+
+data "template_file" "lab_partner_values" {
+  template = file("${path.module}/lab-partner-values.tpl")
+
+  vars = {
+    cluster         = var.cluster
+    namespace       = var.namespace
```

Comment:
You shouldn't need to pass the namespace into the chart as a value because you can use .Release.namespace